### ISSUE TITLE
Error state when refreshing an RSS feed with no include filter matches?

### ISSUE THREAD

**Issue by bpapa9013** [erickok/transdroid#352]
Not entirely sure if this is a bug or not, most likely just a minor UI/logic issue or an unhandled result case, but I am using an RSS feed with ~6 include matches (separated by pipe-bars | as instructed), and when the RSS refreshes with zero matches Transdroid returns '-1' on the count for that RSS feed and acts as though the RSS failed to load at all e.g. when you click the RSS feed the error message "Sorry, please wait until the RSS feed is loaded." is displayed.

I believe the the display for the RSS should just be zero and no error message displayed on click.

To diagnose this I removed the include matches string I'm using and the RSS loaded no problem with many results, just none that matched any of my include match strings. 

When I first setup this RSS there were two items in the RSS feed that matched my include match strings and the RSS loaded correctly and only showed those two matches.

Totally not a big deal, but I thought I would report it.
Thanks,
B~

**Comment by erickok** [comment]
Sounds like a bug.

**Comment by bpapa9013** [comment]
Thanks for looking into it!

I've submitted probably hundreds of bug reports professionally and personally and I think this is like the second time I've received a response from a dev/maintainer/admin/etc...

Thanks for your work on this app, it is a critical and much-loved part of my personal torrenting ecosystem!

**Comment by erickok** [comment]
Ouch that's bad... I was already ashamed I was so slow...

**Comment by bpapa9013** [comment]
Well, TBH I think 90% of the reports I've submitted were to long defunct projects with no active development. I seem to be a sucker for trying to implement orphaned software that I end up having to fix/modify for my own needs/scenarios.

I'm just happy to get any response really, I think the only other direct response I ever got was a "that's not a bug, you're just doing it wrong" sort of grouchy put-off. Don't remember what software that was on tho...

I just updated the transdroid install on my DD phone last night and the only other bug I'd noticed previously was fixed (clicking on RSS results didn't add the torrent to server).
